Framework7/features/step_definitions/get_comments_steps/get_comments

```diff
@@ -0,0 +1,4 @@
+require_relative '../../../src/helpers/rest_client/api_rest_client'
```

Comment:
This file should contains something important in the code and not only requires or require_relative unless that you are using this file to load all the libs that you are going to use. If you are not including something important it should not be uploaded